src/main/kotlin/ro/hibyte/betting/controller/CompetitionController.kt

```diff
@@ -19,7 +20,7 @@ import ro.hibyte.betting.service.CompetitionService
 
 @CrossOrigin(origins = ["http://localhost:4200"])
 @RestController
-@RequestMapping("/api/v1/competitions")
+@RequestMapping("/api/v2/competitions")
```

Comment:
No need to increase the version number yet, since the application didn't reach a production stage yet and there are no real clients that use it